mlflow/server/js/src/components/MetricsPlotPanel.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Utils from '../utils/Utils';
+import RequestStateWrapper from './RequestStateWrapper';
+import { getMetricHistoryApi, getUUID } from '../Actions';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { MetricsPlotView } from './MetricsPlotView';
+import { getRunTags } from '../reducers/Reducers';
+import { MetricsPlotControls, X_AXIS_RELATIVE, X_AXIS_STEP } from './MetricsPlotControls';
+import qs from 'qs';
+import { withRouter } from 'react-router-dom';
+import Routes from '../Routes';
+
+export const CHART_TYPE_LINE = 'line';
+export const CHART_TYPE_BAR = 'bar';
+
+class MetricsPlotPanel extends React.Component {
+  static propTypes = {
```

Comment:
I know we're not super consistent about this in MLflow, but would you mind adding a docstring comment for each of these props? e.g. see [this file](https://github.com/mlflow/mlflow/blob/2a2b2bba3fa2f4c8acfd460e4f3cd7f305ba9c33/mlflow/server/js/src/components/ExperimentView.js#L71) for example